packages/esm-appointments-app/src/homepage-tile/appointments.resources.tsx

```diff
@@ -0,0 +1,16 @@
+import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+import dayjs from 'dayjs';
+
+const useAppointmentsData = () => {
```

Comment:
Duplication of hook from [appointments-tile.resources.tsx](https://github.com/openmrs/openmrs-esm-patient-management/pull/1075/files#diff-b3cb0be0f7940bc5caea96a8895aaec3596f0c06aee555e4ea992496c31836d6)